### PR TITLE
Update AboutUs Page_Diwu-Yi

### LINKED ISSUE #1: Update AboutUs page of the project website

### PR DESCRIPTION
**Pull request by Diwu-Yi**
Update AboutUs Page - Diwu-Yi section by adding photo and modify relevant links and pages.

### CODE REVIEW

File: docs/team/zhangyi.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Zhang Yi's Project Portfolio Page
+---
+
+## Project: tCheck
+
+tCheck is a desktop application used by bubble tea store managers for keeping inventories and keep information of his/her employees. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
```
Comment (ureshiiYing) [inline docs/team/zhangyi.md:17]: I think this page will be continually updated, so we should include what we have done to the project so far - which would be the documentation section.

**Review by ureshiiYing — COMMENTED** [review]
Other than the part in the Project Portfolio Page, the rest looks good to merge.

File: docs/team/zhangyi.md (hunk 1 above)
Comment (Persdre) [reply docs/team/zhangyi.md:17]: I agree :)

Comment (Diwu-Yi) [reply docs/team/zhangyi.md:17]: Noted, will edit accordingly.

**Pull request merged**